packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -374,6 +396,10 @@ export class DocumentDeltaConnection
 			return;
 		}
 
+		if (this.trackLatencyTimer !== undefined) {
+			clearInterval(this.trackLatencyTimer);
```

Comment:
Can you please test manually that timers are reset on disconnects (using examples)?
Anything we can do to automate such validation?

The issue with timers is that if we leak them, they will add up, and eventually tabs will be burning CPU cycles and impact battery life, and it's extremely hard to notice these things.

I do not know if such capability exists, but it might be interesting to see if there is some Node API that checks number of timers in the system, and we could assert at the end of the test that there are none. Or even better - if we use mocks (fake timers), does this library has capability to track active timers?

Reply:
i tested manually and it does reset, i also added a line to reset it to undefined. 
I didn't find any existing API about number of timers